From f1fed3567700ffcc6649568a063df643093edb9e Mon Sep 17 00:00:00 2001
From: Michael Holzheu <holzheu@linux.vnet.ibm.com>
Date: Thu, 17 Jan 2019 17:53:58 +0100
Subject: [PATCH 2/2] lapi/syscalls/s390x: Fix (u)getrlimit syscall numbers

On s390x we only have the getrlimit() syscall with number 191 and no
ugetrlimit() syscall.

Because of the wrong definition the testcase "getrlimit03" fails on s390x.

Signed-off-by: Michael Holzheu <holzheu@linux.vnet.ibm.com>
Acked-by: Jan Stancek <jstancek@redhat.com>
---
 include/lapi/syscalls/s390x.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/lapi/syscalls/s390x.in b/include/lapi/syscalls/s390x.in
index 401943892..e156b0903 100644
--- a/include/lapi/syscalls/s390x.in
+++ b/include/lapi/syscalls/s390x.in
@@ -259,7 +259,7 @@ geteuid 49
 getegid 50
 setreuid 70
 setregid 71
-getrlimit 76
+getrlimit 191
 getgroups 80
 setgroups 81
 fchown 95
@@ -271,7 +271,6 @@ getresuid 165
 setresgid 170
 getresgid 171
 chown 182
-ugetrlimit 191
 mmap2 192
 truncate64 193
 ftruncate64 194
-- 
2.19.1

